common/common.go

```diff
@@ -276,7 +276,6 @@ func CopyI18nStringInfoMap(i18nStringInfoMap map[string]I18nStringInfo) map[stri
 func GetTemplatedStringArgs(aString string) []string {
 	re, err := getTemplatedStringRegexp()
 	if err != nil {
-		fmt.Errorf("i18n4go: Error compiling templated string Regexp: %s", err.Error())
```

Comment:
What is the `go vet` error you are seeing due to this (essentially) log. Is this from a new version. Can you cut and paste and include the version of `go` and `go vet` you are using?

Thanks.